try:
    a=5
    b='python'
    c=a+b
except TypeError:
    print("TypeError Exception occured")
else:
    print("success")
#VallueError
try:
    print(float("python"))

except ValueError:
    print("ValueError occured")
else:
    print("no error")

class Attribute(object):
    pass
try:
    object=Attribute()
    print(object.Attribute)
except AttributeError:
    print("AtrributeError occured")
